deployment/config.go

```diff
@@ -31,6 +31,10 @@ type Config struct {
 	AWSAvailabilityZone string `default:"us-east-1c"`
 	// AWSAMI is the AMI to use for all EC2 instances.
 	AWSAMI string `default:"ami-0fa37863afb290840"`
+	// OperatingSystem
+	OperatingSystemKind string `default:"debian" validate:"oneof:{debian,rhel}"`
+	// AWSAMIUser is the user to use when connecting to the AMI.
+	AWSAMIUser string `default:"ubuntu"`
```

Comment:
Can we avoid adding this one and rely exclusively on OperatingSystemKind? So we get `ubuntu` from `debian`  and whatever RHEL users from `rhel`.

Reply:
It's not that simple, since Ubuntu uses `ubuntu` while Debian uses `admin`. CentOS uses `ec2-user`. I don't know what RHEL (or others) use.

Reply:
Ok, fair enough.

Reply:
Thanks for the context :)